m4/geany-gtkdoc-header.m4

```diff
@@ -21,7 +21,7 @@ AC_DEFUN([GEANY_CHECK_GTKDOC_HEADER],
 	      [test "x$geany_enable_gtkdoc_header" != "xno"],
 	[
 		dnl python
-		AM_PATH_PYTHON([2.7], [have_python=yes], [have_python=no])
+		AM_PATH_PYTHON([3], [have_python=yes], [have_python=no])
```

Comment:
I see what the patch does. But WHY?

Reply:
I agree the commit message could be a bit clearer.
But, @kugel-, I really don't understand why it isn't obvious that we want to use Python3 for build related scripts and to drop Python2 which *IS* dead, since almost two years now.
Is there any reason why we should keep using Python2 for build related scripts?

Reply:
No. I solely object to the lack of *any* background information in the commit message and PR summary. I couldn't care less about python2.

Reply:
Its explained in #2896, but I didn't link it because I didn't want to close it automatically since this PR only does one part of that issue.